### PR TITLE
[OneExplorer] Show and open config on create 

### PR DESCRIPTION
**Pull request by dayo09**
This commit expand the node in the OneExplorer when it is created
and also opens an editor.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

---

This commit conflicts with #745, so it is queued by including the one.

### CODE REVIEW

**Comment by dayo09** [comment]
### GIF

[link](https://github.sec.samsung.net/RS7-RuntimeNTools/secret/issues/187#issuecomment-1533109)

**Comment by yunjayh** [comment]
Q) Do you know that if you press escape key in
image
this situation, a file `undefined` is generated? Is this intended or just a bug to resolve?

**Comment by dayo09** [comment]
@yunjayh That was a bug. Thanks for pointing it out :-D

File: src/OneExplorer/OneExplorer.ts [hunk 1]
```diff
@@ -276,18 +276,30 @@ input_path=${modelName}.${extName}
...
-            console.error(err);
+          if (!value) {
+            Logger.debug('OneExplorer', 'User hit the excape key!');
             return;
           }
 
-          vscode.workspace.fs.writeFile(vscode.Uri.file(cfgPath), content);
+          const uri = vscode.Uri.file(`${dirPath}/${value}`);
+
+          // 'uri' path is not occupied, assured by validateInputPath
```
Comment (yunjayh) [inline src/OneExplorer/OneExplorer.ts:286]: :+1:

**Review by yunjayh — APPROVED** [review]

**Review by hyunsik-yoon — APPROVED** [review]

**Pull request merged**